bitchatTests/GeohashBookmarksStoreTests.swift

```diff
@@ -4,43 +4,45 @@ import XCTest
 final class GeohashBookmarksStoreTests: XCTestCase {
     let storeKey = "locationChannel.bookmarks"
 
+    private let storage = UserDefaultsKeyStorableStub()
+    private var sut: GeohashBookmarksStore?
+    
     override func setUp() {
         super.setUp()
-        // Clear persisted state before each test
-        UserDefaults.standard.removeObject(forKey: storeKey)
-        GeohashBookmarksStore.shared._resetForTesting()
-    }
-
-    override func tearDown() {
-        // Clean after each test
-        UserDefaults.standard.removeObject(forKey: storeKey)
-        GeohashBookmarksStore.shared._resetForTesting()
```

Comment:
If you're not using this function, might as well remove it?

Reply:
I was referring to the actual code of the `_resetForTesting()` in `GeohashBookmarksStore`:

```
    #if DEBUG
    /// Testing-only reset helper
    func _resetForTesting() {
        bookmarks.removeAll()
        membership.removeAll()
        bookmarkNames.removeAll()
        persist()
        persistNames()
    }
    #endif
```